packages/redux/src/raffles/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,264 @@
+import * as actionTypes from '../actionTypes';
+import {
+  mockCreateRaffleParticipationsNormalizedPayload,
+  mockFetchRafflesNormalizedPayload,
+  mockRaffleEstimationResponse,
+  mockRaffleParticipationResponse,
+  mockRaffleResponse,
+  participationId,
+  raffleId,
+} from 'tests/__fixtures__/raffles';
+import reducer, {
+  entitiesMapper,
+  getAllRaffles,
+  getEstimations,
+  getParticipationCreations,
+  getParticipations,
+  getRaffles,
+  INITIAL_STATE,
+} from '../reducer';
+import type { RafflesState } from '../types';
+
+let initialState: RafflesState;
+
+describe('raffles reducer', () => {
```

Comment:
Do not forget to test when the reducer receives `undefined` as its previous state (meaning it is the first time the reducer is invoked).

Reply:
done, pls double check